qcoverage/src/org/qcmg/coverage/Options.java

```diff
@@ -189,13 +154,25 @@ boolean hasLogOption() {
 	boolean hasLogLevelOption() {
 		return options.has("loglevel");
 	}
-
+	
+	//???
+	boolean hasTxtFlag() {
+		 
+		//default (empty) is TXT, or specify TXT
+		return outputFormat.length == 0 || 
```

Comment:
this is confusing - why is the outputFormat a list? Should it not just be a single value?

Reply:
it allows multi-output format,  eg. "--output-format XML --output-format VCF".  I tested it several times, it allows multi-output, It is used to be "--vcf  --xml".

Reply:
So if you supply both `--output-format XML --output-format VCF` options, which format is the output in?

Reply:
it will output both xml and vcf files